packages/grafana-ui/src/components/Combobox/useComboboxFloat.ts

```diff
@@ -35,8 +37,11 @@ export const useComboboxFloat = (
       boundary: document.body,
     }),
     size({
-      apply({ availableWidth }) {
-        setPopoverMaxWidth(availableWidth);
+      apply({ availableWidth, availableHeight }) {
+        const width = Math.max(availableWidth - 16, 0);
```

Comment:
one more minor nit - could probably extract this magic 16 out into a constant, `POPOVER_PADDING` or something? 🤔